src/dds/toolchain/from_dds.cpp

```diff
@@ -570,14 +570,21 @@ toolchain dds::parse_toolchain_dds(const lm::pair_list& pairs, strv context) {
         string_seq ret;
         if (is_msvc) {
             ret = {get_compiler(language::cxx), "/nologo", "/EHsc", "<IN>", "/Fe<OUT>"};
-        } else if (is_gnu_like) {
+        } else if (is_gnu) {
             ret = {get_compiler(language::cxx),
                    "-fPIC",
                    "-fdiagnostics-color",
                    "<IN>",
                    "-pthread",
                    "-lstdc++fs",
```

Comment:
The `-lstdc++fs` is a bit of a goof that should be fixed in a separate context. I'm leaving this note for later.